src/WebJobs.Script/Config/ScriptHostConfiguration.cs

```diff
@@ -135,5 +129,10 @@ public ScriptHostConfiguration()
         /// Gets or sets a test hook for modifying the configuration after host.json has been processed.
         /// </summary>
         internal Action<ScriptHostConfiguration> OnConfigurationApplied { get; set; }
+
+        /// <summary>
+        /// Gets or sets the <see cref="HostHealthMonitorConfiguration"/> to use.
+        /// </summary>
+        public HostHealthMonitorConfiguration HostHealthMonitor { get; set; }
```

Comment:
Do we need a setter?

Reply:
Technically not I guess, but all the other object properties have setters.

Reply:
Removed